L.Control.Range.js

```diff
@@ -21,17 +23,17 @@ L.Control.Range = L.Control.extend({
         slider.step = this.options.step;
         slider.value = this.options.value;
 
-        L.DomEvent.on(slider, 'mousedown mouseup click', L.DomEvent.stopPropagation);
+        L.DomEvent.on(slider, 'mousedown mouseup click touchstart', L.DomEvent.stopPropagation);
 
         /* IE11 seems to process events in the wrong order, so the only way to prevent map movement while dragging the
          * slider is to disable map dragging when the cursor enters the slider (by the time the mousedown event fires
          * it's too late becuase the event seems to go to the map first, which results in any subsequent motion
          * resulting in map movement even after map.dragging.disable() is called.
          */
-        L.DomEvent.on(slider, 'mouseenter', function(e) {
+        L.DomEvent.on(slider, 'mouseenter touchstart', function(e) {
```

Comment:
This is an IE11 work-around. Is the added `touchstart` necessary here, of only above on line 26?

Reply:
Good question! I should have done at least some minor testing before actually adding that.
When testing the demo on iOs the map moved when dragging, but the slider didn't move at all, not by clicking, sliding or whatever. I suspected that map dragging could occur in iOs anyway. I can give it a shot with testing on iPad (does not have an iPhone) without that addition and see if the plugin works as expected. I'll be back asap.